public/static/docs/api-reference/get_url.md

```diff
@@ -0,0 +1,85 @@
+# dvc.api.get_url()
+
+Return the URL to the storage location of the target data (`repo`/`path`).
+
+## Signature
+
+```py
+get_url(path, repo=None, rev=None, remote=None)
+```
+
+## Description
+
+Returns the full URL string (`str` type) to the physical location (in a
+[DVC remote](/doc/command-reference/remote)) where a target <abbr>data
+artifact</abbr> specified by its `path` in a `repo` (<abbr>DVC project</abbr>)
+is stored.
+
+⚠️ Note that the returned URL is formed by analyzing the corresponding
+[DVC-file](/doc/user-guide/dvc-file-format) (see [Examples](#examples) below).
+**There is no guarantee that the file actually exists in that location**. Please
+keep this in mind when using the URL string in your code.
+
+💡 Having the resource's URL, it should be possible to download it directly with
+an appropriate tool such as
+[`urlretrieve`](https://docs.python.org/3/library/urllib.request.html#urllib.request.urlretrieve)
+or `boto3`
+[download_fileobj](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Object.download_fileobj).
+
+Note that for directories, DVC stores a special text file with `.dir` that
```

Comment:
not cleat what can I get from this? Should we give more information how does it actually handle directories?

Reply:
This paragraph is a heads up that if the target is a dir, the URL will end in `.dir` and if you actually download the file, you won't get the data directory itself, but rather a JSON file that is internal to how DVC handles dirs. And yes, I realize all this is not clearly conveyed currently, I'll rewrite... ⏳

Reply:
Changed to 
```markdown
Note that if the target is a directory, the URL will end in `.dir`, as DVC
stores a special JSON file with `.dir` extension that contains the mapping of
files in the directory (as a JSON array), along with their checksums. Refer to
[Structure of cache directory](/doc/user-guide/dvc-files-and-directories#structure-of-cache-directory)
and `dvc add` to learn more about how DVC handles data directories.
```